src/.eslintrc.js

```diff
@@ -63,7 +63,7 @@ module.exports = {
           },
           {
             pattern:
-              "(analytics|components|constants|context|gql|hoc|hooks|pages|types|utils)/**",
+              "{analytics,components,constants,context,gql,hoc,hooks,pages,test_utils,types,utils}{,/**}",
```

Comment:
This change actually breaks the `import/order` plugin. See my comment here.
 https://github.com/evergreen-ci/spruce/pull/1252#discussion_r867010584

![image](https://user-images.githubusercontent.com/4605522/167181566-aefcc17d-1315-4af7-b9b6-45329cdcd0af.png)